docs/pages/setup/reference/config.mdx

```diff
@@ -507,6 +507,25 @@ ssh_service:
     # Enables/disables TCP forwarding. Default is 'true'
     port_forwarding: true
 
+    # When enabled, User's with the "permit_x11_forwarding" role option
+    # will be able to request x11 forwarding sessions with "tsh ssh -X".
+    #
+    # x11 forwarding will only work if the server has the "xauth" binary installed
+    # and supports unix sockets, e.g. "$TEMP/.X11-unix/X[display_number]".
+    #
+    # display_offset and max_display can be used to set a specific range
```

Comment:
Should we move the descriptions of `display_offset` and `max_display` above the relevant config fields?